src/Filter/Setting/CustomSql.php

```diff
@@ -504,40 +486,87 @@ private function parseInsertTagsInternal($queryString)
     /**
      * Replace all insert tags in the query string.
      *
-     * @param string $arrMatch The match from the preg_replace call.
+     * @param string $strMatch The match from the preg_replace call.
      *
      * @return string
      *
      * @internal Only to be used internal as callback from parseSecureInsertTags().
      */
-    public function parseAndAddSecureInsertTagAsParameter($arrMatch)
+    public function parseAndAddSecureInsertTagAsParameter($strMatch)
```

Comment:
Can be made private now and the `@internal` can be removed. This method is now being called from `$this->checkTag()`.